scss/_patterns_side-navigation.scss

```diff
@@ -190,6 +195,15 @@
   .p-side-navigation__toggle,
   .p-side-navigation__toggle--in-drawer {
     @extend %vf-button-base;
+    @include vf-button-pattern(
```

Comment:
In previous theming we only applied theme to `.p-side-navigation__toggle--in-drawer`, not the `.p-side-navigation__toggle`. I guess it would be better to keep it the same for now.

Passing theme colours into the `.p-side-navigation__toggle` button does not work exactly as expected. And I think we can address this later when we properly fix theming in the buttons.